requirements/node.txt

```diff
@@ -0,0 +1 @@
+nodejs-bin
```

Comment:
If we set this to `nodejs-bin[cmd]` that should put `npm` and `node` on the PATH but we'll need to call out a note re: collisions.

Alternatively we can probably intercept the PATH at some point during the installation sequence and CLI invocation shims in this PR and point to the version provided by `nodejs-bin` within our site-packages

See https://pypi.org/project/nodejs-bin/